lib/src/transaction/council_node/unjail_transaction.spec.ts

```diff
@@ -116,11 +118,25 @@ describe('UnjailTransactionBuilder', () => {
         });
     });
 
+    describe('toUnsignedHex', () => {
+        it('should return raw tx hex', () => {
+            const builder = new UnjailTransactionBuilder({
+                stakingAddress: SAMPLE_STAKING_ADDRESS,
+                nonce: SAMPLE_NONCE,
+                network: Mainnet,
+            });
+
+            expect(builder.toUnsignedHex().toString('hex')).to.eq(
+                '01010100000000000000001a642f0e3c3af545e7acbd38b07251b3990914f1002a0100000000000000',
```

Comment:
Above we were concatenating "0102" as the first part of resultant Buffer, here the first 4 chars are "0101". 
Is my understanding bit distorted here?

Reply:
They are different transaction builder, the one you commented before is the `node_join`, and this one is `unjail`.